product_operating_unit/tests/test_product_operating_unit.py

```diff
@@ -39,8 +37,12 @@ def setUp(self):
                                           [self.group_stock_user],
                                           self.company2,
                                           [self.b2b])
+        self.product1.categ_id.operating_unit_ids = [(6, 0, [self.ou1.id])]
         self.product1.operating_unit_ids = [(6, 0, [self.ou1.id])]
```

Comment:
If doing the changes I proposed before, this line is not needed, as the ous in the product should come from the category.